packages/compass-components/src/components/file-input.tsx

```diff
@@ -16,21 +16,21 @@ import {
 
 const { base: redBaseColor } = uiColors.red;
 
-const formItemHorizontalStyles = css({
+const containerStyles = css({
```

Comment:
Move the file name inside the container, the other change here is that the X button to remove the file had to be realigned and the filename inside the button is truncated (before went outside of the button)